### PR TITLE
reading json values to properties

### PR DESCRIPTION
**Pull request by azhamn**

### CODE REVIEW

File: com.wso2telco.dep.hub.creditapi/.project [hunk 1]
```diff
@@ -5,8 +5,14 @@
...
+		<buildCommand>
+			<name>org.eclipse.m2e.core.maven2Builder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
 	</buildSpec>
 	<natures>
+		<nature>org.eclipse.m2e.core.maven2Nature</nature>
 		<nature>org.wso2.developerstudio.eclipse.mavenmultimodule.project.nature</nature>
 	</natures>
```
Comment (rajithsiriwardana) [inline com.wso2telco.dep.hub.creditapi/.project:17]: These change are for credit API. please check what are the file you want to add for a commit. 
additionally you don't need these changes to add.

File: com.wso2telco.dep.hub.customerinfoapi/.project [hunk 2]
```diff
@@ -5,8 +5,14 @@
 	<projects>
 	</projects>
 	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.m2e.core.maven2Builder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
```
Comment (rajithsiriwardana) [inline com.wso2telco.dep.hub.customerinfoapi/.project:12]: Same goes here as well.  I assume you are not intended to have these changes.

File: com.wso2telco.dep.hub.paymentapi/com.wso2telco.dep.hub.paymentapi.synapse/src/main/synapse-config/api/paymentAPI.xml [hunk 3]
```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <api context="/payment" name="paymentAPI" xmlns="http://ws.apache.org/ns/synapse">
   <resource methods="POST" protocol="http" uri-template="/{endUserId}/transactions/amount">
-    <inSequence/>
+    <inSequence>
+      <sequence  key="com.wso2telco.dep.hub.paymentapi.handle.charge.Sequence"/>
+    </inSequence>
     <outSequence/>
     <faultSequence/>
```
Comment (rajithsiriwardana) [inline com.wso2telco.dep.hub.paymentapi/com.wso2telco.dep.hub.paymentapi.synapse/src/main/synapse-config/api/paymentAPI.xml:8]: Add a fault sequnce

File: com.wso2telco.dep.hub.smsmessagingapi/com.wso2telco.dep.hub.ussdapi/.project [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+	<name>com.wso2telco.dep.hub.ussdapi</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+	</buildSpec>
+	<natures>
+		<nature>org.wso2.developerstudio.eclipse.mavenmultimodule.project.nature</nature>
+	</natures>
+</projectDescription>
```
Comment (rajithsiriwardana) [inline com.wso2telco.dep.hub.smsmessagingapi/com.wso2telco.dep.hub.ussdapi/.project:12]: What is ussdapi directory doing inside smsmessagingapi? :O

File: com.wso2telco.dep.hub.smsmessagingapi/com.wso2telco.dep.hub.ussdapi/com.wso2telco.dep.hub.ussdapi.capp/.project [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+			<name>org.eclipse.m2e.core.maven2Builder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.jdt.core.javanature</nature>
+		<nature>org.eclipse.m2e.core.maven2Nature</nature>
+		<nature>org.wso2.developerstudio.eclipse.distribution.project.nature</nature>
+	</natures>
```
Comment (rajithsiriwardana) [inline com.wso2telco.dep.hub.smsmessagingapi/com.wso2telco.dep.hub.ussdapi/com.wso2telco.dep.hub.ussdapi.capp/.project:23]: this folder structure is wrong

File: com.wso2telco.dep.hub.smsmessagingapi/com.wso2telco.dep.hub.ussdapi/com.wso2telco.dep.hub.ussdapi.capp/pom.xml [hunk 6]
```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (rajithsiriwardana) [inline com.wso2telco.dep.hub.smsmessagingapi/com.wso2telco.dep.hub.ussdapi/com.wso2telco.dep.hub.ussdapi.capp/pom.xml:2]: is this ussd changes inside smsmessaging API?

**Review by rajithsiriwardana — CHANGES_REQUESTED** [review]

File: com.wso2telco.dep.hub.creditapi/.project (hunk 1 above)
Comment (shazni) [reply com.wso2telco.dep.hub.creditapi/.project:17]: Agreed. Azham, I informed you that your changes should only be for payment API. I still find changes unrelated to payment API. Please resend the PR again.

**Comment by shazni** [comment]
Closing the PR as it contains unrelated changes to payment API

**Pull request closed without merging**